### PR TITLE
Lims 4270

### PR DESCRIPTION
**Pull request by mazenmmbadr**

### CODE REVIEW

File: ui_testing/testcases/basic_tests/test03_orders.py [hunk 1]
```diff
@@ -423,3 +435,70 @@ def test012_duplicate_many_orders(self):
                                                                                                              selected_order_data[
                                                                                                                  'Test Plans']))
             self.assertEqual(selected_order_data['Test Plans'], row_data['Test Plans'])
+            
+    @skip("https://modeso.atlassian.net/browse/LIMS-4782")
+    def test013_update_order_number(self):
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test03_orders.py]: add logs

File: ui_testing/testcases/basic_tests/test03_orders.py [hunk 2]
```diff
@@ -423,3 +435,70 @@ def test012_duplicate_many_orders(self):
...
+        order_no_created = order_no_created.replace("'", '')
+        
+        # filter by the created order number and get the count
+        self.orders_page.filter_by_order_no(filter_text=order_no_created)
+        self.base_selenium.LOGGER.info(' + filter_by_order_no : {}'.format(order_no_created))
+        rows_data = self.order_page.get_table_rows_data()
+        orders_count = len(rows_data)
+        self.base_selenium.LOGGER.info(' + count_of_the_created_orders : {}'.format(orders_count-1))
+
+        # open the last created order to update its number and checking whether it will affect the rest of the orders or not
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test03_orders.py]: check test001_cancel_button_edit_no test case

File: ui_testing/testcases/basic_tests/test03_orders.py [hunk 3]
```diff
@@ -423,3 +435,70 @@ def test012_duplicate_many_orders(self):
...
+        """
+        New: Orders: Table: Update order number Approach:
+        When I update order number all suborders inside it updated it's order number, 
+        and also in the analysis section.
+
+        LIMS-4270
+        """
+
+        # create order with multiple suborders
+        self.orders_page.click_create_order_button()
```
[the remark this answers is not in the document]
Comment (mazenmmbadr) [reply ui_testing/testcases/basic_tests/test03_orders.py]: that's true, but i wanted to have my orders count to be controlled, if i choose order with duplicate copies, i will need to search the records for order with duplicates, and case i just choose a random order and duplicate it, i will have to filter with this order number first to get its specific count before updating to make sure that the order number has been updated successfully, so it is kinda the same.

File: ui_testing/pages/order_page.py [hunk 4]
```diff
@@ -87,17 +87,23 @@ def set_test_unit(self, test_unit):
     def get_test_unit(self):
         return self.base_selenium.get_text(element='order:test_unit').split('\n')[0]
 
-    def create_new_order(self, material_type='', article='', contact='', test_plan='', test_unit=''):
+    def create_new_order(self, material_type='r', article='r', contact='', test_plan='a', test_unit='a', multiple_suborders=0):
```
[the remark this answers is not in the document]
Comment (mazenmmbadr) [reply ui_testing/pages/order_page.py]: i checked and it is still running, but will change them back to normal and use it in my test case only, to guarantee more random data

**Pull request closed without merging**